spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject do
+    @user = User.new(name: 'Jack', photo: 'Tom.png', bio: 'Teacher from DRC')
+    Comment.new(text: 'Hello', author: @user, id: 4)
+  end
+
+  before { subject.save }
+
+  it 'should return the right comments' do
+    expect(subject.text).to eq 'Hello'
+  end
+end
```

Comment:
- [ ] Kindly add a test for the `update_comments_counter` method you have in your Comment model as it is required to test all the custom methods in each model.

Reply:
- [ ] Please kindly make this change suggested by the previous reviewer. `update_comments_counter` is not a private method so you should be able to create a test for it.

Reply:
Seems you will give me a lead on how to do this because.
 I try impenting it  is not working.